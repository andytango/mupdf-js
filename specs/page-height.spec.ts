import {
  forExamplePages,
  loadExampleFile,
  TEST_DPI,
} from "./helpers/example-file";

describe("pageHeight", () => {
  forExamplePages("should return the page height", async (page) => {
    const { mupdf, doc } = await loadExampleFile();
    expect(mupdf.pageHeight(doc, page, TEST_DPI)).toMatchSnapshot();
  });
});
