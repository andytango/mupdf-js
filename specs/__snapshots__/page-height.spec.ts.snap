// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pageHeight Page [1 / 4] - should return the page height 1`] = `792`;

exports[`pageHeight Page [2 / 4] - should return the page height 1`] = `792`;

exports[`pageHeight Page [3 / 4] - should return the page height 1`] = `792`;

exports[`pageHeight Page [4 / 4] - should return the page height 1`] = `792`;
